File: java/1-sequence-struct/division/src/Main.java
/* Write a program that takes two integers and prints the division between them.
Use float or double for the variable and type conversion for the result.
*/

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        // Instantiating object in java for data entry
        Scanner sc = new Scanner(System.in);

        int n1, n2;
        double div;

        System.out.print("Enter the dividend number: ");
        n1 = sc.nextInt();
        System.out.print("Enter the divisor number: ");
        n2 = sc.nextInt();

        div = (double) n1 / n2;
        System.out.printf("\nDividing %d by %d is %.2f\n", n1, n2, div);

    }
}

File: c/1-sequence-struct/division/main.c
/* Write a program that takes two integers and prints the division between them.
Use float or double for the variable and type conversion for the result.
*/

#include <stdio.h>
#include <stdlib.h>

int main()
{
    int n1, n2;
    double div;

    printf("Enter the dividend number: ");
    scanf("%d", &n1);
    printf("Enter the divisor number: ");
    scanf("%d", &n2);

    div = (double) n1 / n2;
    printf("\nDividing %d by %d is %.2lf\n", n1, n2, div);
    return 0;
}
